.github/workflows/ci.yml

```
@@ -3,166 +1,218 @@
+      
+      - name: Start services
+        run: |
+          cd web
+          docker compose up -d
+          sleep 15  # Wait for services to start
+      
+      - name: Run health checks
+        run: |
+          # Check backend health
+          curl -f http://localhost:8000/api/health || exit 1
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)